src/js/components/HeaderBar.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { HeaderBar as UIHeaderBar } from "#PLUGINS/ui-kit-candidates/HeaderBar";
```

Comment:
I am unsure about the ui-kit-candidates approach here. I have these problems:

- For me, it communicates that all other components will never be moved to the ui-kit
- We now have two positions to look for ui components. We had the same at my last company and it leed to a "starvation" of the right place, because every component was "just" moved to the easier to integrate place with a light polish.

Reply:
you are correct and I discussed it with philipp, we need to be careful with this directory and have to maintain it properly. the Idea was that we move everything that sits in there within a short period of time to ui-kit (or not). 

Do you want me to change something or create issues regarding the move?

Reply:
Can you explain to me why we can not add it to the `ui-kit`?

Reply:
@nLight oh - gotcha! I'll fix that.
@DanielMSchmidt we can, and we should, but I wanted to move forward and optimize later. (same thing as in the tables PR - I'll create a Ticket)